generators/php/codegen/src/context/PhpTypeMapper.ts

```diff
@@ -17,6 +17,11 @@ import { AbstractPhpGeneratorContext } from "./AbstractPhpGeneratorContext";
 export declare namespace PhpTypeMapper {
     interface Args {
         reference: TypeReference;
+        /*
+         * By default, we represent enums as strings, with a phpstan phpdoc referencing the generated enum. If this flag is
+         * is true, then we reference the enum type directly.
+         */
+        preserveEnums?: boolean;
```

Comment:
Is this still used? I don't see anywhere that sets it to `true`.

Reply:
nope not anymore! ok if we leave it around in case, or you want it gone?